tests/acceptance/disconnected-generic/disconnected-agent.err

```diff
@@ -0,0 +1,14 @@
+| versions          - semgrep 0.37.0 on Python
+Note that no baseline-ref was passed and detected environment is not Github or Gitlab to will default to perfoming a full scan. If you want semgrep to only report on new findings from a pull request please set baseline-ref
```

Comment:
FWIW: This somewhat looks like a file you will use to compare to actual output; if so, you have a few spelling errors to correct upstream 🤷 :
```
s/or/nor/
s/to will/so will/
s/perfoming/performing/
```
(Same in push-agent.err below.)